Lib/test/test_os.py

```diff
@@ -2165,6 +2165,14 @@ def test_12084(self):
         finally:
             os.chdir(orig_dir)
 
+    @unittest.skipUnless(os.path.lexists(r'C:\Users\All Users')
+                            and os.path.exists(r'C:\ProgramData'),
+                            'Test directories not found')
+    def test_29248(self):
+        target = os.readlink(r'C:\Users\All Users')
```

Comment:
I would add the explanation from the tracker issue (why we're using this link instead of creating a fresh one to test) here as a comment.

Reply:
done